projects/demo/src/modules/app/stackblitz/project-files/src/main.ts.md

```diff
@@ -1,21 +1,35 @@
 ```ts
-import './polyfills';
+import 'zone.js';
 
-import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
+import {importProvidersFrom} from '@angular/core';
+import {bootstrapApplication} from '@angular/platform-browser';
+import {provideAnimations} from '@angular/platform-browser/animations';
+import {TUI_SANITIZER, TuiDialogModule, tuiSvgOptionsProvider} from '@taiga-ui/core';
+import {TuiPushModule} from '@taiga-ui/kit';
+import {NgDompurifySanitizer} from '@tinkoff/ng-dompurify';
 
-import {AppModule} from './app/app.module';
+import {App} from './app/app.component';
 
-platformBrowserDynamic()
-  .bootstrapModule(AppModule)
-  .then(ref => {
-    // Stackblitz: Ensure Angular destroys itself on hot reloads.
-    if (window['ngRef']) {
-      window['ngRef'].destroy();
-    }
-
-    window['ngRef'] = ref;
-
-    // Otherwise, log the boot error
-  })
-  .catch(err => console.error(err));
+bootstrapApplication(App, {
+  providers: [
+    provideAnimations(),
+    importProvidersFrom(TuiDialogModule, TuiPushModule),
```

Comment:
Why do we need it ?

https://github.com/taiga-family/taiga-ui/blob/db69593859b518ed68c33cd52bc9a6025068aef4/projects/core/components/dialog/dialog.module.ts#L10-L20

https://github.com/taiga-family/taiga-ui/blob/db69593859b518ed68c33cd52bc9a6025068aef4/projects/kit/components/push/push.module.ts#L15-L27